cmd/snap-repair/runner.go

```diff
@@ -318,12 +340,36 @@ func (run *Runner) readState() error {
 	return dec.Decode(&run.state)
 }
 
+func (run *Runner) moveTimeLowerBound(t time.Time) {
+	if t.After(run.state.TimeLowerBound) {
+		run.state.TimeLowerBound = t.UTC()
```

Comment:
Nice :-)

Should we store such lower bound in the state persistently, so that we prevent general time rewind attacks (of whatever kind?)